Makefile

```
@@ -33,79 +36,56 @@ verify:
 unprepublish:
 	sed s/"dist\/"/"src\/"/ packages/tc-ui/package.json > tmp && mv tmp packages/tc-ui/package.json
 
+clean-deps: unprepublish
+	rm -rf packages/*/node_modules
+	rm -rf node_modules
+	rm package-lock.json
+	npm install
 
 # note that this invokes npm install, and in package.json there is a postinstall script
 # defined too, which installs all the node_modules for the packages
 install: unprepublish
 
-prepublish:
-	babel packages/tc-ui/src -D --out-dir packages/tc-ui/dist
-	sed s/"src\/"/"dist\/"/ packages/tc-ui/package.json > tmp && mv tmp packages/tc-ui/package.json
-
-monorepo-publish: prepublish
-	npx athloi version --concurrency 10 $(CIRCLE_TAG)
-	npx athloi publish --concurrency 10 -- --access public
-
-.PHONY: test
 
-.PHONY: cypress-open
-cypress-open: ## cypress-open: Opens the Cypress.io Electron test runner. Expects a local application server to be running concurrently.
-	TREECREEPER_TEST=true TREECREEPER_SCHEMA_DIRECTORY=example-schema \
-	cypress open
-
-.PHONY: cypress-run-page
-cypress-run-page:
-	TREECREEPER_TEST=true TREECREEPER_SCHEMA_DIRECTORY=example-schema \
-	cypress run --spec "packages/tc-ui/src/pages/**/__tests__/**.cyp.js"
-
-.PHONY: cypress-run-primitives
-cypress-run-primitives:
-	TREECREEPER_TEST=true TREECREEPER_SCHEMA_DIRECTORY=example-schema \
-	cypress run --spec "packages/tc-ui/src/primitives/**/__tests__/**.cyp.js"
-
-.PHONY: cypress-verify
-cypress-verify:
-	TREECREEPER_TEST=true TREECREEPER_SCHEMA_DIRECTORY=example-schema \
-	cypress verify
-
-deploy-aws:
-	aws cloudformation deploy --stack-name biz-ops-kinesis --template-body file://$(shell pwd)/aws/cloudformation/biz-ops-kinesis.yaml
-
-test:
-	@if [ -z $(CI) ]; \
-		then NEO4J_BOLT_URL=${LOCAL_BOLT_URL} TREECREEPER_TEST=true TREECREEPER_SCHEMA_DIRECTORY=example-schema DEBUG=true TIMEOUT=500000 \
-			jest --config="./jest.config.js" "${pkg}.*__tests__.*${spec}.*.spec.js" --testEnvironment=node --watch; \
-		else NEO4J_BOLT_URL=${LOCAL_BOLT_URL} TREECREEPER_TEST=true TREECREEPER_SCHEMA_DIRECTORY=example-schema \
-			jest --config="./jest.config.js" "__tests__.*/*.spec.js" --testEnvironment=node --maxWorkers=1 --ci --reporters=default --reporters=jest-junit --detectOpenHandles --forceExit; \
-	fi
-
-run-app: build-statics
-	NEO4J_BOLT_URL=${LOCAL_BOLT_URL} TREECREEPER_TEST=true TREECREEPER_SCHEMA_DIRECTORY=example-schema nodemon --inspect demo/api.js
+# Building and running
 
+## Builds browser files for tc-ui (in watch mode when local)
 build-statics:
 	@if [ -z $(CI) ]; \
-		then $(info Webpack bundling modules ...) ./node_modules/.bin/webpack --mode=development; \
+		then $(info Webpack bundling modules ...) ./node_modules/.bin/webpack --mode=development --watch; \
```

`npx webpack`?

---

just webpack will do as node_modules should be in path
```suggestion
		then $(info Webpack bundling modules ...) webpack --mode=development --watch; \
```